api.proto

```diff
@@ -254,6 +276,11 @@ service API {
     // resumed when they are published to via the Liftbridge Publish API.
     rpc PauseStream(PauseStreamRequest) returns (PauseStreamResponse) {}
 
+    // SetStreamReadonly sets a read-only flag to a partition. The latest
+    // message's offset is returned. Returns a NoSuchStream error code if the
```

Comment:
> The latest message's offset is returned.

Thinking this must have been copied from somewhere else?

Reply:
Ah, no this was an idea we had but didn't implement in the end. I have fixed the comment.